ci/build.sh

```diff
@@ -4,12 +4,22 @@ set -e -u -x -o pipefail
 
 cd "$(dirname "$0")/.."
 
+SPATIALOS_TOOLBELT_VERSION="latest"
 PLATFORM=""
+BINARY=""
 if [[ "$(uname -s)" == "Linux" ]]; then
   PLATFORM="linux"
+  BINARY="spatial"
 elif [[ "$(uname -s)" == "Darwin" ]]; then
   PLATFORM="macos"
+  BINARY="spatial"
 else
   PLATFORM="windows"
+  BINARY="spatial.exe"
 fi
-spatial build --target $PLATFORM --log_level=debug
+
+
+curl -Ls -o "${BINARY}.tmp" "https://console.improbable.io/toolbelt/download/${SPATIALOS_TOOLBELT_VERSION}/${PLATFORM}" || exit 1
+chmod +x "${BINARY}.tmp"
+mv "${BINARY}.tmp" "${BINARY}"
+./$BINARY build --target $PLATFORM --log_level=debug
```

Comment:
`./` not needed.

Reply:
Ping :)

Reply:
It's needed to avoid having to add the directory to `PATH` on Windows.

Reply:
Changed to `spatial` which is a wrapper script I've put on the PATH (needed because we have CMake rules using `spatial`)